### ISSUE TITLE
Tigroid fundus

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#11354]
**Preferred term label:**
Tigroid fundus

**Synonyms**
 Tessellated fundus
Mosaic fundus
Fundus tesselation

**Definition (free text, please give PubMed ID)*
Enhanced visibility of large choroidal vessels through the retinal pigment epithelium (RPE). This reticulated pattern is commonly observed on fundus examination and color fundus photography.  Such hypopigmented areas alternate with hyperpigmented zones in a tigroid pattern. Fundus tessellation is often associated with myopia and choroidal thinning.

Comment: FT may be a normal variant or an early phenotypic indicator of various retinal or choroidal diseases. In pathological conditions such as high myopia, age-related choroidal atrophy, or inherited retinal dystrophies, FT may reflect underlying structural alterations in the choroid, RPE, or sclera, some of which may predispose to significant visual impairment

PMID: 40914444

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
PMID:26908613, POMGNT1, individuals MOGL2063 and SRF1105

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6001439
- label: Tigroid fundus
- parents: [Abnormal fundus morphology (HP:0001098)](https://hpo.jax.org/app/browse/term/HP:0001098)
- pmid: PMID:40914444
- def: Enhanced visibility of large choroidal vessels through the retinal pigment epithelium (RPE). This reticulated pattern is commonly observed on fundus examination and color fundus photography. Such hypopigmented areas alternate with hyperpigmented zones in a tigroid pattern. Fundus tessellation is often associated with myopia and choroidal thinning.
- comment: Tigroid fundus may be a normal variant or an early phenotypic indicator of various retinal or choroidal diseases. In pathological conditions such as high myopia, age-related choroidal atrophy, or inherited retinal dystrophies, FT may reflect underlying structural alterations in the choroid, RPE, or sclera, some of which may predispose to significant visual impairment
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11354
- synonyms: Fundus tesselation(NONE)|Mosaic fundus(NONE)|Tessellated fundus(NONE)
